doc/manuals/_templates/layout.html

```diff
@@ -0,0 +1,15 @@
+{% extends "!layout.html" %}
+  {% block footer %} {{ super() }}
+
+  <style>
```

Comment:
Would a comment here noting that this is for changing the bars to Kitware colors be helpful?

Reply:
Yup